Sources/IdentifiedCollections/IdentifiedArray/IdentifiedArray+Initializers.swift

```diff
@@ -66,53 +66,27 @@ extension IdentifiedArray {
     )
   }
 
-  /// Creates a new array from the elements in the given sequence.
+  /// Creates a new array from the elements in the given sequence, using a combining closure to
+  /// determine the element for any elements with duplicate identity.
   ///
   /// You use this initializer to create an array when you have an arbitrary sequence of elements
-  /// that may not have unique ids. It's safe to pass a sequence with duplicate ids to this initializer,
-  /// later duplicated elements will be discarded.
+  /// that may not have unique ids. This initializer calls the `combine` closure with the current
+  /// and new elements for any duplicate ids. Pass a closure as `combine` that returns the element
+  /// to use in the resulting array: The closure can choose between the two elements, combine them
+  /// to produce a new element, or even throw an error.
   ///
   /// - Parameters:
-  ///   - elements: A sequence of elements to use for the new array. Every key in `elements`
-  ///     must be unique.
+  ///   - elements: A sequence of elements to use for the new array. Every element in `elements`
+  ///     must have a unique id.
```

Comment:
I think I missed this, the elements in this parameter don't have to be unique since that's the whole point of it.